src/System.ServiceModel.Primitives/ref/System.ServiceModel.Primitives.cs

```diff
@@ -1355,6 +1394,7 @@ public sealed partial class ClientOperation
         public ClientOperation(System.ServiceModel.Dispatcher.ClientRuntime parent, string name, string action) { }
         public ClientOperation(System.ServiceModel.Dispatcher.ClientRuntime parent, string name, string action, string replyAction) { }
         public string Action { get { return default(string); } }
+        public System.Collections.Generic.SynchronizedCollection<FaultContractInfo> FaultContractInfos { get { return default(System.Collections.Generic.SynchronizedCollection<FaultContractInfo>); } }
```

Comment:
Nit: can we add a 'using System.Collections.Generic' statement so we don't have to fully qualify types from that namespace?